### PR TITLE
docs: support for mermaid in mkdocs

### PR DESCRIPTION
**Pull request by MustafaSaber**
Since mermaid is supported in Markdown maybe it makes sense to add mkdocs support so making a graph later would work on both.

https://mermaid.js.org/syntax/flowchart.html?id=flowcharts-basic-syntax

### CODE REVIEW

File: docs/kubernetes/ingress-controller.md [hunk 1]
```diff
@@ -231,11 +231,18 @@ In kubernetes skipper-ingress fetches ingress/routegroup configurations every **
...
+  graph TD;
+      kapis(kubeapiserver) --fetches ingresses--> s(skipper);
+```
+
 
 ### After
 
-![skipper-with-routesrv](../img/skipper_with_routesrv.svg)
+```mermaid
+  graph TD;
```
Comment (MustafaSaber) [inline docs/kubernetes/ingress-controller.md:243]: backward arrows are coming(hopefully) https://github.com/mermaid-js/mermaid/pull/4019

**Comment by MustafaSaber** [comment]
image

**Comment by AlexanderYastrebov** [comment]
Let fix the pipeline, otherwise :+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**